### DISCUSSION TITLE
Mod unavailable on Workshop - Force Download

### DISCUSSION THREAD

**Discussion by Yordanorn** [soda3x/ArmaReforgerServerTool#55] [Q&A]
Hi,

First, I'm sorry about my english, I'm french and I'm not a good user of english ^^'

I find this powerfull Tool and it is awesome !
But I have a problem with a mod.

I create my settings for my server and Unfortunately One mod is unavailable on workshop (It's a recent problem).
I delete the mod on my config in the pannel but when I launch the start, the tool try to download this mod again and this error stop the work.

You know how can I found a solution please ?

**Comment by soda3x** [comment]
oh it tries to download the mod even if it's not in Enabled Mods? Interesting... maybe some caching is happening, I will have a look into it, thanks for using the tool!
